bin/build_requirements

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ruby
+require 'json'
+require 'vanagon/extensions/ostruct/json'
+require 'vanagon/extensions/set/json'
+require 'vanagon/extensions/hashable'
+
+load File.expand_path(File.join(File.dirname(__FILE__), "..", "lib", "vanagon.rb"))
+
+optparse = Vanagon::OptParse.new("#{File.basename(__FILE__)} <project-name> <platform-name> [options]", %i[workdir configdir engine])
+options = optparse.parse! ARGV
+
+project = ARGV[0]
+platform = ARGV[1]
+
+unless project or platform
```

Comment:
shouldn't this be `unless (project and platform)`?  like, you want this warning to come up if either are unset, yeah?

Reply:
(parens may not be necessary, but I put 'em in for clarity)

Reply:
oh, yeah, I think I copied that from another bin and it was wrong.